packages/driver/test/cypress/integration/commands/navigation_spec.coffee

```diff
@@ -567,6 +567,15 @@ describe "src/cy/commands/navigation", ->
               "http://localhost:3500/fixtures/dimensions.html"
             ])
 
+    ## https://github.com/cypress-io/cypress/issues/1311
+    it "doesn't invoke onLoad callback when visiting the same URL with hashes", ->
```

Comment:
we can now rename to refer directly to the behavior of resolving immediately without attempting to change the URL or cause a reload